<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgRestextrakt</class>
 <widget class="QDialog" name="DlgRestextrakt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Berechnung Exktrakt</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Spindel</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Extrakt</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="DoubleSpinBox" name="spinBox_S_SwPlato">
          <property name="minimumSize">
           <size>
            <width>67</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>67</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>99.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>20.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_18">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>째P</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Dichte</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="DoubleSpinBox" name="spinBox_S_SwDichte">
          <property name="minimumSize">
           <size>
            <width>67</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>67</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_14">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>째C</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Temperatur</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="DoubleSpinBox" name="spinBox_S_Temperatur">
          <property name="minimumSize">
           <size>
            <width>67</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>67</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>4.000000000000000</double>
          </property>
          <property name="maximum">
           <double>99.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>20.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_19">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>g/ml</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Refraktometer</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QLabel" name="label_BrixRefraktometer">
          <property name="text">
           <string>Extrakt</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QComboBox" name="comboBox_FormelBrixPlato">
          <item>
           <property name="text">
            <string>Terrill</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Terrill linear</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Standardformel</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kleier</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="DoubleSpinBox" name="spinBox_R_SwBrix">
          <property name="minimumSize">
           <size>
            <width>67</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>67</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>99.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>20.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_FormelBrixPlato">
          <property name="text">
           <string>Umrechnungsformel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_EinheitBrix">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>째Brix</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Korrekturfaktor </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="DoubleSpinBox" name="tbKorrekturFaktor">
          <property name="minimumSize">
           <size>
            <width>67</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>67</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1.050000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
          <property name="value">
           <double>1.030000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Extrakt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DoubleSpinBox" name="spinBox_SwPlato">
       <property name="minimumSize">
        <size>
         <width>67</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>67</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>20.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>째P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>widgets/doublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinBox_S_SwPlato</tabstop>
  <tabstop>spinBox_S_SwDichte</tabstop>
  <tabstop>spinBox_S_Temperatur</tabstop>
  <tabstop>comboBox_FormelBrixPlato</tabstop>
  <tabstop>spinBox_R_SwBrix</tabstop>
  <tabstop>spinBox_SwPlato</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgRestextrakt</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgRestextrakt</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
